### DISCUSSION TITLE
Custom styles for components per theme

### DISCUSSION THREAD

**Discussion by Dennise89** [vuetifyjs/vuetify#17814] [General]
Hello!

I can set global styles for components like this:
`export default createVuetify({
  defaults: {
    VBtn: {
      class: 'text-none',
    },
  },
})`

But this will be for dark and light theme. Is there also a possibility to set the global styles per theme?

**Comment by jervalles** [comment]
I was wondering the same. As I wanna edit the background color of every toolbars per theme.

Like on default theme: background white,
And on dark theme, the default color of the component

But I don't wanna edit every single v-tool-bar

**Comment by craigrileyuk** [comment]
Agreed. You should be able to set a "defaults" property on each theme which is then merged with the globals.

**Comment by Saury-t** [comment]
If I set a pink theme, I'm not quite clear about this aspect. What is the scope of influence for this theme? I see that the documentation doesn't provide detailed instructions.
